### DISCUSSION TITLE
DOC: Mention that monitor control is available as a home-brew cask

### DISCUSSION THREAD

**Discussion by corneliusroemer** [MonitorControl/MonitorControl#1282] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I wanted to quickly check what monitor control is called in home-brew but couldn't find any mention of it in the README to my surprise.

It is available as `brew install --cask monitorcontrol` - as expected - but it would be nice if you briefly referenced this somewhere? After all maybe people will like the convenience of installing from the command line.

### Describe the solution you'd like

Add a line mentioning that monitor control can be installed through `brew install --cask monitorcontrol` as well

### Describe alternatives you've considered

-

**Comment by waydabber** [comment]
Hi @corneliusroemer - can you post a PR regarding this? If properly texted, I'll approve it. Thanks! :)
